R/utilities.R

```diff
@@ -127,19 +127,113 @@ get_ssh_session = function(host="gphost01.bcgsc.ca"){
 #'
 gene_to_region = function(gene_symbol,
```

Comment:
This function can no longer return the region in the original chr:start-end format, right?  Why was that functionality removed rather than configuring the function to default to that behaviour unless otherwise specified? I think that restoring the deleted line 139 would actually solve this.

Reply:
If you run this function with the default parameter for return_as, you get the original format chr:start-end. this parameter was added so that you can specify "df" to get regions returned as a data frame in R (for improved compatibility with other functions that can take a df with regions/genes, e.g the new subset parameter in fancy_ideogram).

Reply:
But the default is bed now, isn't it? That's not the same format as chr:start-end

Reply:
My bad, what do we prefer to call this format then?

Reply:
Would help to see what it looks like. Can you show me the format of what's returned when you run with default arguments right now? I have been referring to this format as just "region". Bed is a specific tab-delimited format but a bed-compatable data frame could also reasonably be called "bed".